ext/mysqli/mysqli_fe.c

```diff
@@ -261,6 +261,7 @@ ZEND_END_ARG_INFO()
 ZEND_BEGIN_ARG_INFO_EX(arginfo_mysqli_query, 0, 0, 2)
 	MYSQLI_ZEND_ARG_OBJ_INFO_LINK()
 	ZEND_ARG_INFO(0, query)
+	ZEND_ARG_INFO(0, resultmode)
```

Comment:
Changing the name of the parameter is not a problem, from a technical point of view. However, the docs would need to be updated. Imho this is worth it, having this kind of grammar issues in code is not a good sign for quality.